plugins/node/opentelemetry-instrumentation-express/examples/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "express-example",
+  "private": true,
+  "version": "0.23.0",
+  "description": "Example of Express integration with OpenTelemetry",
+  "main": "index.js",
+  "scripts": {
+    "zipkin:server": "cross-env EXPORTER=zipkin ts-node src/server.ts",
+    "zipkin:client": "cross-env EXPORTER=zipkin ts-node src/client.ts",
+    "jaeger:server": "cross-env EXPORTER=jaeger ts-node src/server.ts",
+    "jaeger:client": "cross-env EXPORTER=jaeger ts-node src/client.ts",
+    "compile": "tsc -p ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/open-telemetry/opentelemetry-js.git"
+  },
+  "keywords": [
+    "opentelemetry",
+    "express",
+    "tracing"
+  ],
+  "engines": {
+    "node": ">=8"
+  },
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
+  },
+  "dependencies": {
+    "@opentelemetry/api": "^1.0.2",
+    "@opentelemetry/exporter-jaeger": "^0.26.0",
+    "@opentelemetry/exporter-zipkin": "^0.26.0",
+    "@opentelemetry/instrumentation": "^0.27.0",
+    "@opentelemetry/instrumentation-http": "^0.27.0",
+    "@opentelemetry/resources": "^0.26.0",
+    "@opentelemetry/sdk-trace-base": "^0.26.0",
```

```suggestion
    "@opentelemetry/sdk-trace-base": "^1.0.0",
```
same below for sdk-trace-node. Otherwise some packages like e.g @opentelemetry/core are installed twice.

---

Could you please update all stable core Otel components to ^1.0.0? Mixing versions likely wont work and it is for sure not what we want that people do.